test/CodeGen/AVR/sign-extension.ll

```diff
@@ -0,0 +1,11 @@
+; RUN: llc -march=avr < %s | FileCheck %s
+
+define i64 @sign_extended(i1) {
+; CHECK-LABEL: entry-block
+entry-block:
+  %1 = sext i1 %0 to i8
+  %2 = sext i8 %1 to i16
```

Comment:
oh wow that's ugly

It's probably not worth testing this seeing that - not really any way the logic could go wrong.